worlds/animal_well/region_data.py

```diff
@@ -442,15 +442,17 @@ class AWData(NamedTuple):
         rname.bear_chameleon_room_1:
             AWData(AWType.region),
         lname.candle_bear:
-            AWData(AWType.location, [[iname.bubble, iname.matchbox], [iname.disc, iname.matchbox]],
+            AWData(AWType.location, [[iname.bubble, iname.matchbox], [iname.disc, iname.matchbox],
+                                     [iname.wheel_hop, iname.matchbox]],
                    loc_type=LocType.candle),
         lname.candle_bear_event:
-            AWData(AWType.location, [[iname.bubble, iname.matchbox], [iname.disc, iname.matchbox]],
+            AWData(AWType.location, [[iname.bubble, iname.matchbox], [iname.disc, iname.matchbox],
+                                     [iname.wheel_hop, iname.matchbox]],
                    event=iname.event_candle_bear),
         lname.fruit_49:
-            AWData(AWType.location, [[iname.bubble], [iname.disc]], loc_type=LocType.fruit),
+            AWData(AWType.location, [[iname.bubble], [iname.disc], [iname.matchbox]], loc_type=LocType.fruit),
         lname.fruit_50:
-            AWData(AWType.location, [[iname.bubble], [iname.disc]], loc_type=LocType.fruit),
+            AWData(AWType.location, [[iname.bubble], [iname.disc], [iname.matchbox]], loc_type=LocType.fruit),
```

Comment:
```suggestion
            AWData(AWType.location, [[iname.bubble], [iname.disc], [iname.wheel_hop]], loc_type=LocType.fruit),
```
I'm assuming this was the intent, yeah?

Reply:
Zero idea how this happened lmao